src/main/java/uk/gov/hmcts/reform/sscs/service/BulkPrintService.java

```diff
@@ -116,6 +118,62 @@ private static Map<String, Object> getAdditionalData(final SscsCaseData sscsCase
         additionalData.put(LETTER_TYPE_KEY, "sscs-data-pack");
         additionalData.put(CASE_IDENTIFIER, sscsCaseData.getCcdCaseId());
         additionalData.put(APPELLANT_NAME, sscsCaseData.getAppeal().getAppellant().getName().getFullNameNoTitle());
+        additionalData.put(RECIPIENTS, getPartiesOnTheCase(sscsCaseData));
         return additionalData;
     }
+
+    private static List<String> getPartiesOnTheCase(SscsCaseData caseData) {
+        log.info("Getting recipients on the case for additional data for caseId {}", caseData.getCcdCaseId());
+        List<String> parties = new ArrayList<>();
+        Appeal appeal = caseData.getAppeal();
+        Appellant appellant = appeal.getAppellant();
+
+        parties.add(appellant.getName().getFullNameNoTitle());
```

Comment:
appellant.getName().getFullNameNoTitle()
potential NPE on getName. 
You are checking it in below condition but not here.

Please refactor

Reply:
I thought that appellant full name can never be null. In line 120, there was no check to get the appellant name, so I went something similar here. But I am happy to add a condition.